projects/packages/scheduled-updates/tests/php/class-scheduled-updates-test.php

```diff
@@ -43,6 +43,7 @@ public static function set_up_before_class() {
 	protected function set_up() {
 		parent::set_up_wordbless();
 		\WorDBless\Users::init()->clear_all_users();
+		Scheduled_Updates::init();
```

Comment:
@TimBroddin Do you remember what made this necessary?

Reply:
As far as I remember, it needed to be called because otherwise the tests regarding plugin deletion would fail because this action wasn't registered:

```
add_action( 'deleted_plugin', array( __CLASS__, 'deleted_plugin' ), 10, 2 );
```

Slack context: p1711976808791809/1711965300.902539-slack-C01A60HCGUA

Reply:
Thanks!